lua/starfall/libs_cl/render.lua

```diff
@@ -902,7 +904,11 @@ end
 -- @param number b Number, blue value
 -- @param number a Number, alpha value
 function render_library.setRGBA(r, g, b, a)
-	currentcolor = Color(r, g, b, a)
+	-- currentcolor = Color(r, g, b, a)
+	rawset(currentcolor, "r", r or 255)
+	rawset(currentcolor, "g", g or 255)
+	rawset(currentcolor, "b", b or 255)
+	rawset(currentcolor, "a", a or 255)
```

Comment:
Not tested but I think this introduces a bug. This causes `currentcolor` to be modified in place rather than replacing it. This could cause `COLOR_WHITE` (default value of `currentcolor`) as well as the argument to a previous call of `render.setColor` to become altered unexpectedly.

Reply:
What could go wrong? It is not used anywhere else except for setting the initial color.